website/docs/guides/core-to-cloud-1.md

```diff
@@ -0,0 +1,244 @@
+---
+title: Move from dbt Core to dbt Cloud - Getting started
+id: core-to-cloud-1
+description: "Learn how to move from dbt Core to dbt Cloud and what you need to get started."
+hoverSnippet: "Use this guide to learn how to move from dbt Core to dbt Cloud."
+icon: 'guides'
+time_to_complete: 'Total estimated time: 3-4 hours'
+hide_table_of_contents: true
+tags: ['Migration','dbt Core','dbt Cloud']
+level: 'Intermediate'
+recently_updated: true
+---
+## Introduction
+
+Moving from dbt Core to dbt Cloud streamlines analytics engineering workflows by allowing teams to develop, test, deploy, and explore data products using a single, fully managed service.
```

Comment:
Did we get stakeholders to sign-off on describing it as a "fully managed service"? We sell services separately and that line kind of indicates that maybe those services are included.
```suggestion
Moving from dbt Core to dbt Cloud streamlines analytics engineering workflows by allowing teams to develop, test, deploy, and explore data products using a single, fully managed service.
```

Reply:
i think this might have come from the team internally. i can double check with @azzam34

Reply:
confirmed with PMM that this is fine since service in this context is software as a service. will add the word software to clarify.